fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
digest is definitely a requirement, since the implementation plan says we will record the checksum in the event triples: https://wiki.duraspace.org/display/FF/Audit+Service+Implementation+Proposal#AuditServiceImplementationProposal-RDFVocabulary

We haven't included size, but it is the other factor that's checked in the fixity check, so I can see including it too.